### PR TITLE
 Add docs for dir resource and datasource

### PR DESCRIPTION
**Pull request by SimonBarendse**

### CODE REVIEW

File: docs/resources/secret.md [hunk 1]
```diff
@@ -15,17 +15,25 @@ This resource allows you to write secrets at a given path, if the path already e
 To write a secret:
 
 ```terraform
+resource "secrethub_dir" "repo" {
```
Comment (florisvdg) [inline docs/resources/secret.md:18]:
```suggestion
data "secrethub_dir" "repo" {
```

File: docs/resources/secret.md [hunk 2]
```diff
@@ -15,17 +15,25 @@ This resource allows you to write secrets at a given path, if the path already e
...
-  path  = "company/repo/ssh_key"
+  path  = "${data.secrethub_dir.repo.path}/ssh_key"
   value = file("/path/to/ssh/key")
 }
 ```
 
 To generate a new, 20 characters long secret made of alphanumeric characters:
 
 ```terraform
+resource "secrethub_dir" "repo" {
```
Comment (florisvdg) [inline docs/resources/secret.md:31]:
```suggestion
data "secrethub_dir" "repo" {
```

File: docs/resources/secret.md [hunk 3]
```diff
@@ -36,8 +44,12 @@ resource "secrethub_secret" "db_password" {
 To generate a new secret made of lowercase letters and symbols, with minimum 5 symbols:
 
 ```terraform
+resource "secrethub_dir" "repo" {
```
Comment (florisvdg) [inline docs/resources/secret.md:47]:
```suggestion
data "secrethub_dir" "repo" {
```

**Comment by SimonBarendse** [comment]
Good spot @florisvdg! I've applied your suggestions.

With these changes applied, is this ready to go? I'm proposing to merge to `master`, so that we can immediately release to get these directory additions visible in the registry.

**Review by florisvdg — APPROVED** [review]

**Pull request merged**